inc/command/class-command.php

```diff
@@ -213,11 +214,18 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 	 */
 	protected function codecept( InputInterface $input, OutputInterface $output ) {
 		$options = $input->getArgument( 'options' );
-		$tests_folder = $input->getOption( 'path' );
+		// Codeception command runs within `vendor` directory, so paths are relative to that.
+		$tests_folder = '../' . $input->getOption( 'path' );
 		$output_folder = $input->getOption( 'output' );
-		$run_headless_browser = $input->getOption( 'browser' );
+		$run_headless_browser = $input->getOption( 'browser' ) ?: 'chrome';
```

Comment:
Why not leave this as the default value for the option now that starting the browser is dependent on whether the suite uses the web driver module?